docs/deploy/self-hosted/k8s.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Kubernetes
+---
+
+The ParadeDB Helm Chart is based on the official [CloudNativePG Helm Chart](https://cloudnative-pg.io/). CloudNativePG is a Kubernetes operator that covers the full lifecycle of a highly available
+PostgreSQL database cluster with a primary/standby architecture, using native streaming replication.
+
+The chart is also available via [ArtifactHub](https://artifacthub.io/packages/helm/paradedb/paradedb).
+
+## Getting Started
+
+First, install [Helm](https://helm.sh/docs/intro/install/). The following steps assume you have a Kubernetes cluster running v1.25+. If you are testing locally, we recommend
+using [Minikube](https://minikube.sigs.k8s.io/docs/start/).
+
+### Installing the CloudNativePG Operator
+
+Skip this step if the CNPG operator is already installed in your cluster.
+
+```console
+helm repo add cnpg https://cloudnative-pg.github.io/charts
+helm upgrade --install cnpg \
+--namespace cnpg-system \
+--create-namespace \
+cnpg/cloudnative-pg
+```
+
+### Setting up a ParadeDB CloudNativePG Cluster
+
+Create a `values.yaml` and configure it to your requirements. Here is a basic example:
+
+```yaml
+type: paradedb
+mode: standalone
+
+cluster:
+  instances: 2
+  storage:
+    size: 256Mi
+```
+
+You can omit the `--values values.yaml` to use the default values. Please refer to the [ParadeDB Helm Chart documentation](https://paradedb.github.io/charts/) for advanced configuration options.
+
+Then, launch the ParadeDB cluster.
+
+```console
+helm repo add paradedb https://paradedb.github.io/charts
+helm upgrade --install paradedb \
+--namespace paradedb-database \
+--create-namespace \
+--values values.yaml \
+paradedb/paradedb
+```
+
+## High Availability
+
+<Info>
```

Comment:
Do we still want to add this banner? Right now it's not actually closed source, but the plan is to make some of it closed source.